### DISCUSSION TITLE
Redirect 'give us feedback' link exactly to this page, instead of Github issues

### DISCUSSION THREAD

**Discussion by mredigonda** [vercel/turborepo#7166] [Ideas]
Currently, in https://turbo.build/pack/docs, when clicking on "Question? Give us feedback →", you're taken to:

https://github.com/vercel/turbo/issues/new/choose

I propose to link here instead: https://github.com/vercel/turbo/discussions/new?category=ideas, or some other much more simplified way of providing feedback, as those other templates end up requiring you to send "Link to code that reproduces this issue", or "What package manager are you using / does the bug impact?", which don't really apply to a problem in docs.

**Comment by anthonyshew** [comment]
This is great idea! Fixing this in #7216.
